mslice/util/qt.py

```diff
@@ -0,0 +1,21 @@
+def load_ui(caller_filename, ui_relfilename, baseinstance=None):
+    from qtpy.uic import loadUi, loadUiType
```

Comment:
Is there a reason you have these imported inside the function?

Reply:
Not really, I'l move them outside so it's consistent.